models/pendo__feature_daily_metrics.sql

```diff
@@ -0,0 +1,73 @@
+with spine as (
```

Comment:
One question and comment on this model. Is a feature restricted to one version at a time? If so, I think it would be a good add here to include the version in the daily metrics table to allow for some neat A/B testing of the feature versions over time. 

We can chat more about this live.

Reply:
We chatted about this and comfortable resolving the conversation.